csrc/fips_status.cpp

```diff
@@ -17,13 +18,16 @@ extern "C" void AWS_LC_fips_failure_callback(char const* message);
 #if defined(FIPS_SELF_TEST_SKIP_ABORT)
 void AWS_LC_fips_failure_callback(char const* message)
 {
-    const size_t char_limit = 128;
+    fprintf(stderr, "AWS_LC_fips_failure_callback invoked with message: '%s'\n", message);
+    // Must track
+    // https://github.com/aws/aws-lc/blob/e4885d5e22f7dc482dd6bfa713b0e1b763b5c538/crypto/fipsmodule/self_check/self_check.c#L52
```

Comment:
Ideally, AWS-LC would expose this value in a header value somewhere and ACCP would use that value, but this is fine for now.

Reply:
agreed.